### DISCUSSION TITLE
Advice / Suggestions / Insights for IT Collaboration

### DISCUSSION THREAD

**Discussion by VinArse** [microsoft-search/pnp-modern-search#3766] [General]
I asked chat GPT how I should call the people of the GitHub community, in a funny way....
"How about "CodeClowns" or "GitGiggles"? They add a playful touch to the serious world of coding!"
and a serious way,,,
"For a more serious tone, you could go with "CodeCrafters" or "TechTacticians." These names convey a sense of professionalism and expertise in the world of coding and development."
naa! Don't like any of them ;)
I’m a project manager, so I don’t know much about the world of coding….my advanced apologies if I’m unclear. 

Hello all,

**Contexte:**
We are using pnp modern search on our intranet. IT department added to the SharePoint. A technical genius in our team has configured it. We are using it with category/goods/services searches for procurement. We love the app. Works GREAT. Really great.  We have been using it in beta mode and everything is perfect.

**Challenge:**
Our IT department: they want to unplug it ☹ They say they don’t have the manpower to maintain it.
I think they would have the following enquiries:
1-	How frequently is it updated? 
2-	How complex are the updates? (it’s manual)
3-	How many man hours to update?
4-	What if we don’t update it?
5-	What companies use this open-source app? How popular is it ?
But since they are busy, (I don’t want to bother them) I’m looking for some answers to provide all this to them.

**Proposed Solution:**
I’m thinking my technical genius, and I can keep track of needed updates, document changes and do as much as possible on our side and at a certain frequency, would ask IT to update pnp (we don’t know the effort behind this) I would ask IT if we could update it ourselves, but I’m thinking this would be risky for them and an access/security issue.

**your insight**
Do you have answers to my questions 1 to 5?
What do you think of my proposed solution?

Thank you very much for any insight you may have. If you can answer only some or part of my questions ;)

Cheers!

**Comment by kasperbolarsen** [comment]
Hi,
1- How frequently is it updated? **_Hard to say as we don't have a fixed schedule. So far we have had 3 releases in 2024_**
2- How complex are the updates? (it’s manual)  **super simple**
3- How many man hours to update?  **15 minutes max**
4- What if we don’t update it?   **We recommend updating from time to time, as some of the updates are security related. Likewise you will only be able to get assistance from the project maintainers on the latest version**
5- What companies use this open-source app? How popular is it ?  **We can't and we will NOT track which companies are using the web parts. However, we know that the web parts are installed on serveral thousand tenants** 

You can install the web part package in specific site collections and thus manage updates etc yourself, see https://microsoft-search.github.io/pnp-modern-search/installation/

**Reply by VinArse** [reply]
Hello,
Thank you **VERY** much for your quick and precise reply.

2 stakeholders in play: 
1- IT : admins of SharePoint platform for the whole company
2- Finance (me and my tech genius) : admins of one site. Where we have installed PnP

You could rightfully tell me "just talk to your IT" ;) But our IT is not available/busy/understaffed. We need to assess how autonous we can be.

A small clarification, you write : 
"You can install the web part package in specific site collections and thus manage updates etc yourself,"
"Yourself", you mean without IT's help ? IT could give us the rights to update PnP ?
Could IT deploy it only to our site ? and make us more autonomous for updates ?

danke!

**Reply by kasperbolarsen** [reply]
I have to assume that you are Site Collection Admin on the site is question, otherwise you can't do anything.
Setting up the Site Collection App Catalog, https://learn.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog

Once it has been set up, you will be able to install new versions of PnP Modern Search web parts, and IT will not have to be involved unless we are asking for additional permissions for the web parts, which we haven't done for quite awhile.

**Comment by VinArse** [comment]
Thank you SO much for all your answers :)
